react/week-2/lesson.md

```diff
@@ -522,26 +535,26 @@ This is a very common pattern which will come in very useful!
 | **Exercise** |
 | :--- |
 | 1. Open the `pokedex` React application again and open the `src/BestPokemon.js` file. |
-| 2. Create a new component called `BestPokemonFetcher`. |
+| 2. Add a new component inside `src/BestPokemon.js` called `BestPokemonFetcher`. |
 | 3. Change the `export default` to export `BestPokemonFetcher` instead of `BestPokemon`. We **don't** need to make any changes to the `BestPokemon` component. |
```

Comment:
Is it weird to have the *default* export (BestPokemonFetcher) not match the file name (BestPokemon)?  Newbie asking.  ( I see that pattern existed prior to this PR)

Reply:
Probably, yes. We could either change the name of the file, or better still, create a new file entirely for the BestPokemonFetcher.

Or, moving it out into a separate component could be an exercise at the end after the 'Container component' section, to illustrate that `BestPokemonFetcher` is a container, separate from `BestPokemon` the presentational component

Reply:
Yes it is explicitly going against the "best practice" that we've asked them to follow here.

I think I was trying to avoid spending time making sure students had correctly linked up separate component files. Also note that this file is further extended in week 3. In hindsight I think having everything in 1 file is a bad idea.

I can make a change which change the instructions if you'd like.

Reply:
FYI I've added a mini exercise after 'Container components' for students to have a think about where we could apply the container/presentational pattern in pokedex - and the 'right answer' would be that `BestPokemonFetcher` is a container while `BestPokemon` is presentational. It's at the very end of the day and if there's time we could get students to split those 2 components then, to implement the pattern?

Reply:
>  It's at the very end of the day and if there's time we could get students to split those 2 components then, to implement the pattern?

Sounds like a good plan 👍